recipes/qscintilla2/bld.bat

```diff
@@ -0,0 +1,73 @@
+@echo off
+:: MSVC library for Python 2.7 is not found automaticaly if it is 
+:: installed in the user's AppData folder. Add it to PATH
+setlocal EnableDelayedExpansion
+
+if "%PY_VER%"=="2.7" (
+	set VC9DIR=%LOCALAPPDATA%\Programs\Common\Microsoft\Visual C++ for Python\9.0
+	IF EXIST !VC9DIR! ( 
+		set PATH=!VC9DIR!;"%PATH%"
+		if %ARCH% == 32 call vcvarsall.bat x86
+		if %ARCH% == 64 call vcvarsall.bat amd64
```

Comment:
I am not sure about this.... Windows is not my forte but I have the impression that `conda-build` should take care of that for you, no?